fg/settings/config.py

```diff
@@ -75,18 +102,18 @@
 
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.postgresql_psycopg2',
-        'NAME': 'postgres',
-        'USER': 'postgres',
-        'HOST': 'db',
-        'PORT': '5432',
+        'ENGINE': getenv('FG_DB_ENGINE',default='django.db.backends.postgresql_psycopg2'),
+        'NAME': getenv('FG_DB_NAME',default='postgres'),
```

Comment:
Nitpick - make sure to put one space after your commas (I used to do this too!)

Reply:
Got it - will do